ale_linters/json/vscodejson.vim

```diff
@@ -10,7 +10,7 @@ endfunction
 call ale#linter#Define('json', {
 \   'name': 'vscodejson',
 \   'lsp': 'stdio',
-\   'executable': 'vscode-json-language-server',
+\   'executable': 'vscode-json-languageserver',
 \   'command': '%e --stdio',
 \   'project_root': function('ale_linters#json#vscodejson#GetProjectRoot'),
 \})
```

Comment:
I just had a read of the issue. Did the executable use the old name previously? We should fall back on the old name if so. The C and C+++ compiler Kinross llinters are good examples of where the default setting is to try different names, but it can also be configured to be a specific name.